docs/easy_install.txt

```diff
@@ -317,8 +317,8 @@ Note that instead of changing your ``PATH`` to include the Python scripts
 directory, you can also retarget the installation location for scripts so they
 go on a directory that's already on the ``PATH``.  For more information see
 `Command-Line Options`_ and `Configuration Files`_.  During installation,
-pass command line options (such as ``--script-dir``) to
-``ez_setup.py`` to control where ``easy_install.exe`` will be installed.
+pass command line options (such as ``--script-dir``) to to control where
```

Comment:
This says 'to to'. I tried to fix it, but when I edit in GitHub, that text isn't present.